### PR TITLE
[WIP] Website with documentation

### PR DESCRIPTION
**Pull request by TOsmanov**
### Motivation and context

This PR contains a website created using an extended version of Hugo 0.81 ([hugo documentation](https://gohugo.io/documentation/)) and a docsy theme ([docsy documentation](https://www.docsy.dev/docs/)) .

- The site contains a homepage, a about page and documentation. The main documentation page duplicates the README.md file. 
- User Guide is divided into several pages for easy reading without having to scroll a long page (issue #1630).
- Added .gitmodule file to sync docsy theme
- Temporary website files added to the .gitignore file

Current issues:
- File "site/assets/icons/logo.svg" is a copy of "cvat-ui/src/assets/cvat-logo.svg" with color changed.
- Links lead to the "develop" branch

### Change table:
 
| Original file                                                                                         | Current file                                                                                                                                                                         | Move | Divide | Rename| Compare | Delete original file|  
|--------------------------------------------------------------------------------- |-------------------------------------------------------------------------------------------------------------------------------------------- |---------|----------|------------|-------------|-------------------------|
|cvat/tree/develop/cvat/apps/documentation/instalation.md | cvat/site/content/en/docs/for-users/instalation.md                                                                                                 |X          |             |                |X                | X                               |
| cvat/tree/develop/cvat/apps/documentation/user_guide.md | cvat/site/content/en/docs/for-users/user-guide/                                                                                                    |X          | X          |                |X                | X                               |
|cvat/tree/develop/cvat/apps/documentation/installation_automatic_annotation.md| cvat/site/content/en/docs/for-users/installation_automatic_annotation.md           |X          |             |                |X                | X                               |
|cvat/tree/develop/cvat/apps/documentation/faq.md               | cvat/site/content/en/docs/for-users/faq.md                                                                                                             |X          |             |                |X                | X                                |
|cvat/tree/develop/components/analytics/README.md            |cvat/site/content/en/docs/for-developers/analytics.md                                                                                          |X          |             |X              |X                | X                               |
|cvat/tree/develop/cvat/apps/dataset_manager/formats/README.md   |cvat/site/content/en/docs/for-developers/formats/                                                                               |X          |X           |X              |X               |X                                |
|cvat/tree/develop/utils/cli/README.md                                      |cvat/site/content/en/docs/for-developers/cli.md                                                                                                      |X          |             |X               |X               |X                                |
|cvat/tree/develop/cvat/apps/documentation/xml_format.md |cvat/site/content/en/docs/for-developers/xml_format.md                                                                                   |X         |              |                 |X               |X                                |
|cvat/tree/develop/cvat/apps/documentation/AWS-Deployment-Guide.md |cvat/site/content/en/docs/for-developers/AWS-Deployment-Guide.md                                     |X         |              |                 |X               |X                                |
|cvat/tree/develop/utils/dataset_manifest/README.md           |cvat/site/content/en/docs/for-developers/dataset_manifest.md                                                                          |X         |              |X               |X               |                                  |
|cvat/tree/develop/cvat/apps/documentation/data_on_fly.md|cvat/site/content/en/docs/for-developers/data_on_fly.md                                                                                   |X         |              |                 |X               |X                                |
|cvat/tree/develop/cvat/apps/documentation/mounting_cloud_storages.md |cvat/site/content/en/docs/for-developers/mounting_cloud_storages.md                               |X         |              |                 |X               |X                                |
|cvat/blob/develop/cvat/apps/documentation/backup_guide.md |cvat/site/content/en/docs/for-developers/backup_guide.md                                                                          |X         |              |                 |X               |                                  |

### The documentation has the following structure:
Documentation
│   _index.md
│
├───for-developers
│   │   analytics.md
│   │   AWS-Deployment-Guide.md
│   │   backup_guide.md
│   │   cli.md
│   │   dataset_manifest.md
│   │   data_on_fly.md
│   │   mounting_cloud_storages.md
│   │   xml_format.md
│   │   _index.md
│   │
│   └───formats
│       │   _index.md
│       │
│       └───format-specifications
│               format-camvid.md
│               format-coco.md
│               format-cvat.md
│               format-icdar.md
│               format-imagenet.md
│               format-labelme.md
│               format-market1501.md
│               format-mot.md
│               format-mots.md
│               format-tfrecord.md
│               format-vggface2.md
│               format-voc.md
│               format-widerface.md
│               format-yolo.md
│               _index.md
│
└───for-users
    │   faq.md
    │   installation.md
    │   installation_automatic_annotation.md
    │   _index.md
    │
    └───user-guide
        │   controls-sidebar.md
        │   creating_an_annotation_task.md
        │   downloading-annotations.md
        │   getting_started.md
        │   interface.md
        │   models.md
        │   objects-sidebar.md
        │   projects.md
        │   search.md
        │   settings.md
        │   task-synchronization.md
        │   top-panel.md
        │   types-of-shapes.md
        │   vocabulary.md
        │   workspace.md
        │   _index.md
        │
        ├───advanced
        │   │   ai-tools.md
        │   │   analytics.md
        │   │   annotation-with-polylines.md
        │   │   annotation-with-rectangle-by-4-points.md
        │   │   annotation-with-tags.md
        │   │   attribute-annotation-mode-advanced.md
        │   │   automatic-annotation.md
        │   │   filter.md
        │   │   opencv-tools.md
        │   │   review.md
        │   │   shape-grouping.md
        │   │   shape-mode-advanced.md
        │   │   shortcuts.md
        │   │   track-mode-advanced.md
        │   │   _index.md
        │   │
        │   ├───annotation-with-cuboids
        │   │       creating-the-cuboid.md
        │   │       editing-the-cuboid.md
        │   │       _index.md
        │   │
        │   ├───annotation-with-points
        │   │       liner-interpolation-with-one-point.md
        │   │       points-in-shape-mode.md
        │   │       _index.md
        │   │
        │   └───annotation-with-polygons
        │           automatic-borders.md
        │           edit-polygon.md
        │           manual-drawing.md
        │           track-mode-with-polygons.md
        │           _index.md
        │
        └───basics
                attribute-annotation-mode-basics.md
                basic-navigation.md
                shape-mode-basics.md
                track-mode-basics.md
                _index.md

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: site/content/en/debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[0316/104620.328:ERROR:registration_protocol_win.cc(130)] TransactNamedPipe: The pipe has been ended. (0x6D)
```
Comment (nmanovic) [inline site/content/en/debug.log:1]: @TOsmanov , it should be deleted I believe.

**Comment by nmanovic** [comment]
@TOsmanov , please add README.md into site directory with basic instructions how to add changes and build a new version of the documentation. Just document your current documentation development life cycle.

File: site/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+## Basic manual for website editing
+
+### Edit or add documentation pages
+
+To edit and/or add documentation, you need to have a [githab](https://github.com/login) account.
```
Comment (azhavoro) [inline site/README.md:5]:
```suggestion
To edit and/or add documentation, you need to have a [GitHub](https://github.com/login) account.
```

File: site/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    title: "Title"
+    linkTitle: "Link Title"
+    weight: 1
+    description: >
+        Description
+    ---
+
+### Start site localy
+
+To start the site locally, you need a recent [extended version hugo](https://github.com/gohugoio/hugo/releases) (recommend version 0.75.0 or later).  Open the most recent release and scroll down until you find a list of Extended versions.
```
Comment (azhavoro) [inline site/README.md:26]: I would suggest to add a link to the installation guide.

File: site/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+Add a path to "hugo" in the "Path" variable environment.
```
Comment (azhavoro) [inline site/README.md:28]:
```suggestion
Add a path to "hugo" in the "Path" environment variable.
```

**Comment by nmanovic** [comment]
@TOsmanov , what was the reason to close the PR?

**Comment by TOsmanov** [comment]
> @TOsmanov , what was the reason to close the PR?

This PR has a lot of wrong commits. I plan to create a new branch and a new PR.

**Pull request closed without merging**